docs/08-verification_pallets.md

```diff
@@ -2,7 +2,65 @@
 title: Verification Pallets
 ---
 
-## Coming soon
+Zkverify provide a set of verifier pallets that expose `submitProof` extrinsic call
+to verify the given proofs and store the proof statement.
 
-## Fflonk
-## Boojum
+Every time that a valid proof is verified the `Zkverify` emit an **event** that identify the
+submitted statement and the `id` that can be used to check proof availability on the
+destination chain (i.e. Ethereum). All verified statements are uniquely identified by the
+hash computed as follows
+
+```text
+hash := Keccak256('<verifier-id>-'<public-input>)
+```
+
+where:
+
+* `verifier-id` is a unique identifier string for what statement we should prove
+* `public-input` are the public input bytes of the proved statement
+
+## Fflonk Verifier
+
+This verifier check if the given proof is a Polygon-CDK Fork-Id 6 valid proof. The input proof format
+is 800 bytes array where the last 32 bytes are the public inputs: the pallet use
+[`fflonk_verifier` crate](https://github.com/HorizenLabs/fflonk_verifier/tree/v0.3.0) to deserialize
+the proof and public inputs and then verify them. If we put error handling aside we can sketch the pallet duty
+with the following code snipped
+
+```rust
+let proof = Proof::try_from(&proof_data).unwrap();
+let pubs = pubs_data.into();
+
+proof.verify(pubs).unwrap()
+```
+
+If the proof is correct a `Poe::NewElement(statement, attestation_id)` event is emitted where `statement`
+is computed by use `fflonk` as `verifier-id`.
+
+This call can fail both if be not possible deserialize the proof (`InvalidProofData`) or proof don't 
+verify (`VerifyError`)
+
+## Zksync Verifier
```

Comment:
Let's put a link to the pallet as well and a reference to the submitProof extrinsic